toolkit/components/uniffi-bindgen-gecko-js/fixtures/generated/SpritesScaffolding.cpp

```diff
@@ -28,13 +28,13 @@ struct Result {
 //
 // For async calls this should be called in the main thread, since the GC can
 // move the ArrayBuffer pointers while the background task is waiting.
-Args PrepareArgs(const JS::Handle<JS::Value>& ptr,
+Args PrepareArgs(const UniFFIPointer& ptr,
```

Comment:
Not directly related to this, but I wonder if the template should generate `[NS_ASSERTION(NS_IsMainThread(),"blah");` (github won't let me add the comment to the template itself :)